crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -14,12 +14,16 @@ use opentelemetry_sdk::{
     trace, Resource,
 };
 use opentelemetry_semantic_conventions::resource;
+use settings::{ConsoleLogSettings, OtlpLogSettings, OtlpTraceSettings};
```

Comment:
nit: This should be moved down (not an external crate).

Reply:
Done in afc21636bf3ec0324aabd3dcf20b65dfe38de946